Rakefile

```diff
@@ -65,7 +65,7 @@ namespace :dependencies do
 
   namespace :bundle do
     task :check do
-      sh 'bundle check --path=${BUNDLE_PATH:-vendor/bundle} > /dev/null', verbose: false do |ok, _res|
+      sh 'bundle check > /dev/null', verbose: false do |ok, _res|
```

Comment:
Using `--path` prints a warning and given we run the command on a Git hook, the warning would have appeared quite often in the terminal...

```
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local path 'vendor/bundle'`, and stop using this flag
```

Notice that we already have the result of the suggested `bundle config set local path` call in our `.bundle/config` file.